### PR TITLE
Adding a wse balancer test

### PR DESCRIPTION
**Pull request by jitsni**
Forward porting the test

### CODE REVIEW

File: service/http.balancer/src/test/java/org/kaazing/gateway/service/http/balancer/WsebBalancerIT.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        }
+    };
+
+
+    @Rule
+    public TestRule chain = outerRule(robot).around(gateway).around(timeout);
+
+    @Test
+    @Specification("wse.balancer.request")
+    public void wseBalancerRequest() throws Exception {
```
Comment (cmebarrow) [inline service/http.balancer/src/test/java/org/kaazing/gateway/service/http/balancer/WsebBalancerIT.java]: Usually we try to make test the names reflect the expected outcome sent immediately obvious what is testing for. How about balancerRequestShouldRespondWithRedirect?

**Comment by cmebarrow** [comment]
Is there a separate request for the actual code fix?

**Comment by cmebarrow** [comment]
@jitsni done reviewing, couple of minor comments/questions

**Comment by jitsni** [comment]
There is no code fix here (only adding tests). Now ready to merge.

**Pull request merged**